gobblin-data-management/src/main/java/org/apache/gobblin/data/management/retention/dataset/MultiVersionCleanableDatasetBase.java

```diff
@@ -322,12 +322,14 @@ public void clean()
   }
 
   private boolean cleanDatasetVersions(List<T> versions, VersionSelectionPolicy<T> selectionPolicy,
-      List<RetentionAction> retentionActions)
+      List<RetentionAction> retentionActions, boolean shouldLogDeletableVersion)
```

Comment:
a better name could have been `runningInBatches` `runningInAnIterator` or something like that. I mean it would be better to name it based on the condition/cause, not the effect we want.

Reply:
Changed to `runningInBatcheMode`